README.md

```diff
@@ -140,7 +140,7 @@ The Github token needs all of the "repo" permissions:
 
 Once you have the token, you can set the secret value to the token like this:
 
-`terminus secret:set ${SITE_NAME} github-oauth.github.com ${GITHUB_TOKEN} --type=composer --scope user --scope ic`
+`terminus secret:set ${SITE_NAME} github-oauth.github.com ${GITHUB_TOKEN} --type=composer --scope user,ic`
```

Comment:
Missing "=" here? Not sure if really needed but may be worth adding it for consistency.

Reply:
Fixed